allvars.h

```diff
@@ -520,7 +520,7 @@ extern struct particle_data
     pthread_spinlock_t SpinLock;
 #endif
 
-    float GravCost;     /*!< weight factor used for balancing the work-load */
+    int GravCost;     /*!< weight factor used for balancing the work-load */
```

Comment:
It must have been there to down weight the high redshift hydro. I have overlooked this. I think it is OK to leave this field float, but do the rounding in domain.c to avoid the problem -- because there we scale the variable by TimeBase, a huge number.